### ISSUE TITLE
Newsletter

### ISSUE THREAD

**Issue by olorton** [phpoxford/ideas#4]
I've been thinking about this, is this still something that we want?

It could be a good way of showing people what we get up to and encourage more people to join. If it was quarterly, then we could send out a summary of the last 3 talks, and advertise any future talks that are booked in, spires updates, hackdays, etc.

How would we run it?
One idea is to create another repo with markdown files for each quarter, then we can add updates throughout the quarter before (the owner) sends it out using mailchimp.

I'm happy to own this if no-one else is interested.

**Comment by olorton** [comment]
@ascii-soup I know that you talked about this a while back, is this the kind of thing that you had in mind? Did you want to own this. I'm happy to if you do not.

**Comment by ascii-soup** [comment]
Yeah I'm happy to own it, no problem. I think it would be good - also a
good way to send updates to people ("Don't forget we've got Oliver giving a
talk on blah next week")

**Comment by olorton** [comment]
Repo created... go wild https://github.com/phpoxford/mailinglist

**Comment by martindilling** [comment]
:+1: for having a newsletter ;)
